test/extensions/filters/http/common/compressor/websocket_with_compressor_integration_test.cc

```diff
@@ -0,0 +1,248 @@
+#include "test/extensions/filters/http/common/compressor/websocket_with_compressor_integration_test.h"
```

Comment:
this (and .h) is copy/pasted test/integration/websocket_integration_test (i've just removed few things which are not related to upgrade and renamed class/structs by adding WithCompressor (as well as added compressor configuration itself))